This patch removes the legacy oslo namespace deprecation check from the
oslo.middleware library. The deprecated() function is left to satisfy
the unit tests. It is not suitable for the upstream and should be
removed once all of the relevant consumers have switched over to the
new namespace.

--- oslo.middleware-0.4.0/oslo/middleware/__init__.py.~1~	2015-01-29 08:34:30.000000000 -0800
+++ oslo.middleware-0.4.0/oslo/middleware/__init__.py	2015-03-02 23:04:40.549834298 -0800
@@ -23,6 +23,3 @@ def deprecated():
         DeprecationWarning,
         stacklevel=3,
     )
-
-
-deprecated()
